lib/csp.js

```diff
@@ -19,8 +37,10 @@ module.exports = function (options) {
         name += '-Report-Only';
     }
 
-    for (key in policyRules) {
-        value += key + ' ' + policyRules[key] + '; ';
+    if (typeof policyRules === 'object') {
```

Comment:
I do not think it will fail.. it will send null to `parsePolicyRules` where nothing happens.